docs/pages/application-access/guides/dynamic-registration.mdx

```diff
@@ -57,6 +57,29 @@ spec:
 
 See the full app resource spec [reference](../reference.mdx#application-resource).
 
+The user's role creating the dynamic registration must have access to the 
```

Comment:
Let's reword this, as it sounds like "the user's role" is the thing creating the registration, when it's the user.

Reply:
updated @zmb3 , please review

The user creating the dynamic registration needs to have a role with access to the...